craft_store/store_client.py

```diff
@@ -24,7 +24,7 @@
 from overrides import overrides
 from pymacaroons.serializers import json_serializer
 
-from . import endpoints, errors
+from . import creds, endpoints, errors
```

Comment:
how about "credentials", bytes are cheaper than ever :-)

Reply:
Generally I agree but in this case I think "creds" is clear enough to still be worth it. Bytes are cheap but my "horizontal scanning real estate" isn't ;)